src/registrar/views/domain.py

```diff
@@ -42,6 +42,15 @@ class DomainView(DomainPermissionView):
 
     template_name = "domain_detail.html"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        security_email = self.get_object().get_security_email()
+        if security_email is None or security_email == "dotgov@cisa.dhs.gov":
```

Comment:
same comment here, pull from domain.getdefaultsecurityEmail or get it from PublicContact instead of hardcoding it